settlement/src/execution.rs

```diff
@@ -50,7 +50,11 @@ pub fn create_execution_transaction(
             format!("ex-transfer-nft: {}", execution.target_chain)
         }
     };
-    let body = serde_spb::to_string(&execution).unwrap();
+    let body = format!(
+        "{}\n---\n{}",
```

Comment:
It automatically converts bytes32 type to string. For example, it converts ```"0x2f8dcc7f5b046a4d47951c33c06f41fa62d7c9939d018251d05da631b8a1e7aa"```
 to 
```"32663864636337663562303436613464343739353163333363303666343166613632643763393933396430313832353164303564613633316238613165376161"```, which is a string in hex.